Dockerfile

```diff
@@ -6,4 +6,7 @@ COPY . /app
 WORKDIR /app
 RUN python3 setup.py install
 
-ENTRYPOINT [ "/usr/local/bin/python3", "example.py" ]
\ No newline at end of file
+#Install requirements
+RUN pip3 install -r requirements.txt
+
+ENTRYPOINT [ "/usr/local/bin/python3", "data_server.py" ]
```

Comment:
I think we'll need a strategy what the entrypoint will be like for the docker image  :-) Maybe create something more versatile that can dispatch to the intended command?
I stayed with the default example when I introduced it so that it can still be used as a "command line tool" for instance.